standalone/portal-api/src/modules/relationship/dto/request/update-relationship.dto.ts

```diff
@@ -1,16 +1,21 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty, IsString, ValidateNested } from 'class-validator';
+import {
+  IsNotEmpty,
+  IsOptional,
+  IsString,
+  ValidateNested,
+} from 'class-validator';
 import { CreatePaymentExperienceRequest } from './create-payment-experience.dto';
 import { Type } from 'class-transformer';
 
 export class UpdateRelationshipRequest {
   @ApiProperty({ example: 'Relationship' })
-  @IsNotEmpty()
+  @IsOptional()
```

Comment:
Is this correct? We had it mandatory before

Reply:
Those two fields aren't required in quick application , they are required to filled in full application, hence optional